tests/metrics/network/dockerfile_image/README.md

```diff
@@ -0,0 +1,12 @@
+# Building the network image 
```

Comment:
Please, do not use headings for numbered steps. Also use the imperative:
Build the network image with the following command: